documents/agent-setup.md

```diff
@@ -0,0 +1,112 @@
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>OID4VC demo's agent setup 
+  <br>
+</h1>
+
+---
+
+**Warning: This project still is in very early development. Breaking changes without notice will happen at this point!**
+
+## Agent Configuration Guide
+
+Welcome to the agent setup guide. This document will walk you through configuring your agent module by leveraging the settings defined in your `.env` file. Specifically, the `CONF_PATH` property in your `.env` file dictates where the agent retrieves its configuration from.
+
+### Configuration Directory Structure
+
+Within your `packages/agent/conf` directory, navigate to the `examples` subfolder. Here, you will find six key directories, each integral to the agent's configuration:
+
+1. **dids**: This directory is designated for storing Decentralized Identifiers (DIDs) documents. These did documents will be used throughout or oid4vci and oid4vp/siop process.
+
+2. **oid4vci_metadata**: In this folder, you'll find JSON files that detail the metadata required by the [VCI specification](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html) and some values that are essential for configuring this agent. Following is a table of values that should be set for a json file inside `oid4vci_metadata`:
+
+| Variable              | Sub-Variable                 | Description                                                                                                                                                                                                     |
+|-----------------------|------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `correlationId`       |                              | A unique identifier used to track the transaction within the system.                                                                                                                                            |
+| `overwriteExisting`   |                              | Determines whether to replace existing data with new data.                                                                                                                                                      |
+| `metadata`            |                              | Contains configuration details and specifications for credential issuance.                                                                                                                                      |
+|                       | `credential_issuer`          | The entity that issues and signs the credential.                                                                                                                                                                |
```

Comment:
entity is way to broad. Please lookup the actual definition in the spec. This is a URL

Reply:
changed to the value in oid4vci spec